### PR TITLE
Notify integration test

### PR DESCRIPTION
**Pull request by sinclairs**
Adds an integration test for the current notify service. This is in preparation for a major version bump to `notifications-node-client`.

Adds a `test:local` npm script which allows the integration test to be excluded.

Update: Now has the notifications-node-client 5.1.0

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -7,12 +7,6 @@ const config = {
 };
 /*! m0-end */
 
-process.env.DCS_JWT_SECRET = '123';
-process.env.DCS_LOG_LEVEL = 'silent';
-process.env.NOTIFY_API_KEY = '123apiKey';
-process.env.WEBHOOK_DEV_REPORTER_SLACK = 'webhook-url';
-process.env.APP_ENV = 'test';
```
Comment (armoj) [inline jest.config.js:14]: Are none of these being used at all in tests?

File: services/notify/index.js [hunk 2]
```diff
@@ -10,19 +10,29 @@ const notifyClient = new NotifyClient(apiKey);
...
+            return {
+                id: smsSendRequest.body.id
+            };
         } catch (err) {
-            logger.error({err}, 'SMS SEND FAILURE');
+            logger.error(err.error, 'SMS SEND FAILURE');
         }
 
-        return response;
+        return undefined;
```
Comment (armoj) [inline services/notify/index.js:35]: would it be worth having the return values in the same shape here, do you think?

**Review by armoj — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 3]
```diff
@@ -69,3 +69,5 @@ typings/
 .vscode
 
 .config
+
+jest.env.js
```
Comment (armoj) [inline .gitignore:73]: just spotted this. Ignore my comment about the env vars

**Review by armoj — CHANGES_REQUESTED** [review]

File: jest.config.js (hunk 1 above)
Comment (sinclairs) [reply jest.config.js:14]: Closed as per: https://github.com/CriminalInjuriesCompensationAuthority/data-capture-service/pull/236#discussion_r757049178

File: services/notify/index.js (hunk 2 above)
Comment (sinclairs) [reply services/notify/index.js:56]: Possibly. I was just maintaining the current behaviour. Nothing uses it at the moment.

File: services/notify/index.js [hunk 4]
```diff
@@ -5,24 +5,55 @@ const createMessageBusCaller = require('../message-bus');
...
     async function sendSms(options) {
-        let response;
         try {
-            response = notifyClient.sendSms(options.templateId, options.phoneNumber, {
-                personalisation: {
-                    case_reference: options.personalisation.caseReference
-                },
-                reference: null
-            });
+            const smsSendRequest = await notifyClient.sendSms(
```
Comment (armoj) [inline services/notify/index.js:33]: `notificationDocument`
`notificationReceipt`

**Pull request merged**